### DISCUSSION TITLE
Plotter Automation

### DISCUSSION THREAD

**Discussion by pieterhelsen** [Chia-Network/chia-blockchain#1632] [General]
Hi guys,

Wanted to pick the community's brain with regards to automating the plotting process. Right now I'm plotting mostly via the GUI, but obviously that becomes a little tedious, so I was wondering if anyone here has any experience with automating their workflow and how they went about doing that. 

Some questions to get the discussion started:

- Do you or would you use a Python-based service to manage the process or do you just time out the plots with a crontab for instance?
- Do you check the progress of other plots to determine how many plots to run in parallel? If so, how do you do that? (as far as I can tell, there's no command line available to check on running plots)
- What about environment monitoring? Did you make it robust by checking your available disk space or any other system parameters?

Would love to hear the community's feedback on this!

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
